commit ea5736a92a07e402dcba4203e1149e17692787be
Author: forlinx <forlinx@gmail.com>
Date:   Tue May 3 14:03:07 2016 +0800

    add led1 led2

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dts b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
index da9a271..57d9587 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
@@ -22,6 +22,21 @@
 	memory {
 		reg = <0x80000000 0x20000000>;
 	};
+	leds {
+	compatible = "gpio-leds";
+	pinctrl-names = "default";
+
+	led1{
+		label = "led1";
+		gpios = <&gpio5 9 GPIO_ACTIVE_LOW>;
+		default-state = "off";
+	};
+	led2{
+		label = "led2";
+		gpios = <&gpio1 9 GPIO_ACTIVE_LOW>;
+		default-state = "off";
+	};
+	};
 
 	backlight {
 		compatible = "pwm-backlight";
@@ -55,8 +70,6 @@
 			regulator-name = "VSD_3V3";
 			regulator-min-microvolt = <3300000>;
 			regulator-max-microvolt = <3300000>;
-			gpio = <&gpio1 9 GPIO_ACTIVE_HIGH>;
-			enable-active-high;
 		};
 
 		reg_gpio_dvfs: regulator-gpio {
@@ -438,7 +451,6 @@
 				MX6UL_PAD_LCD_RESET__WDOG1_WDOG_ANY    0x30b0
 				MX6UL_PAD_UART1_RTS_B__GPIO1_IO19	0x17059 /* SD1 CD */
 				MX6UL_PAD_GPIO1_IO05__USDHC1_VSELECT	0x17059 /* SD1 VSELECT */
-				MX6UL_PAD_GPIO1_IO09__GPIO1_IO09        0x17059 /* SD1 RESET */
 			>;
 		};
 
@@ -537,8 +549,7 @@
 				MX6UL_PAD_LCD_ENABLE__LCDIF_ENABLE  0x79
 				MX6UL_PAD_LCD_HSYNC__LCDIF_HSYNC    0x79
 				MX6UL_PAD_LCD_VSYNC__LCDIF_VSYNC    0x79
-				/* used for lcd reset */
-				MX6UL_PAD_SNVS_TAMPER9__GPIO5_IO09  0x79
+	
 			>;
 		};
 
